src/options.c

```diff
@@ -349,58 +349,60 @@ static const char *getPathOfStdout(void)
 
 void optionsParse(int argc, char *argv[])
 {
-    static char stropts[] = "a:ofipbcd:e:hmq:s::t:uvzn:l:D:k::C:S:F:M:w:";
-
+    int optch;
```

Comment:
Any specific reason for bringing this above the static arrays? Don't really like mixing static and automatic variables.

Reply:
No, will revert.